indico/modules/users/models/users.py

```diff
@@ -277,6 +277,12 @@ class User(PersonMixin, db.Model):
         nullable=False,
         default=False
     )
+    #: the date the user has last accepted the terms of service
+    accepted_tos_date = db.Column(
+        db.DateTime,
```

Comment:
if you kept the time component (didn't check that yet), use our `UTCDateTime` like we use everywhere else. if not, use the native `Date`

Reply:
Ah nice, didn't know about that one, will do. I kept the time component for when the user accepts so there is a specific point in time, but the admin value is just a date which uses the server timezone.